Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9
+
+RUN mkdir -p /app
+
+COPY . main.py /app/
+
+WORKDIR /app
+
+COPY ./requirements.txt /app/requirements.txt
```

Comment:
we don't need to copy requirements.txt again after running "COPY . /app/" from line 4.

Reply:
I have corrected it as recommended